back-end/src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/assignment_submission_db
-spring.datasource.username=example_user
-spring.datasource.password=password123
+spring.datasource.username=root
+spring.datasource.password=Accornupyourbutt25!
```

Comment:
Don't check in your passwords! This is a lesson that I had to learn a lot of times in my career. Good to get started with those lessons now.